azavea/ism-watershed-wellness-snapshot
# Implement sensor details modal

## Overview

Adds the sensor details modal to the app. It is relatively static at this point, besides the name of the sensor, but will be wired up in #41.

The component is implemented as a React Portal, which allows us to insert it at the top level of the app DOM instead of as a child of the overview component so that is can be displayed correctly.

Connects #22

### Demo

![image](https://user-images.githubusercontent.com/1042475/52049617-45b03a80-251c-11e9-9975-52413d2f1389.png)

## Testing Instructions

- Click the map markers to display the sensor overview, then click the list icon to open the details component.
- Verify the correct sensor name is displayed, and that the modal is closed when clicking the "x" button in the top right corner and the close button at the bottom of the modal.
